plugins/become/podman_unshare.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2022 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# Written by Janos Gerzson (grzs@backendo.com)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    name: podman_unshare
+    short_description: Run tasks using podman unshare
+    description:
+        - This become plugins allows your remote/login user
+          to execute commands in its container user namespace.
+          Official documentation: https://docs.podman.io/en/latest/markdown/podman-unshare.1.html
```

Comment:
The colon in this line causes the error:

ERROR: plugins/become/podman_unshare.py:14:33: error: DOCUMENTATION: syntax error: mapping values are not allowed here (syntax)

Reply:
I see, thanks